content/docs/1.5.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -20,6 +20,7 @@ This page covers the following topics:
 - [Enable virtual-hosted-style access for S3 compatible Backupstore](#enable-virtual-hosted-style-access-for-s3-compatible-backupstore)
 - [NFS Backupstore](#nfs-backupstore)
 - [SMB/CIFS Backupstore](#smbcifs-backupstore)
+- [Set up Azure Blob Storage Backupstore](#set-up-azure-blob-storage-backupstore)
```

Comment:
Can you help update `NFS` and `CIFS` section header to `Set up...` as well?

Reply:
No problem, updated.